charms/kfp-ui/requirements.in

```diff
@@ -1,3 +1,5 @@
+# Workaround for https://github.com/python-jsonschema/jsonschema/issues/1117
+# and https://github.com/python-jsonschema/jsonschema/issues/1114
```

Comment:
Is this comment needed?

Reply:
good catch!  Removed the pin, but not the comment.  ty!